storage/storage_test.go

```diff
@@ -68,23 +68,23 @@ func newStore(t *testing.T) storage.Store {
 	}
 	run := filepath.Join(wd, "run")
 	root := filepath.Join(wd, "root")
-	uidmap := []idtools.IDMap{{
+	Transport.SetDefaultUIDMap([]idtools.IDMap{{
 		ContainerID: 0,
 		HostID:      os.Getuid(),
 		Size:        1,
-	}}
-	gidmap := []idtools.IDMap{{
+	}})
+	Transport.SetDefaultGIDMap([]idtools.IDMap{{
 		ContainerID: 0,
 		HostID:      os.Getgid(),
 		Size:        1,
-	}}
+	}})
 	store, err := storage.GetStore(storage.StoreOptions{
 		RunRoot:            run,
 		GraphRoot:          root,
 		GraphDriverName:    "vfs",
 		GraphDriverOptions: []string{},
```

Comment:
Non-blocking: Would it be easy, and safe, to add some values to `GraphDriverOptions`, to test the option parsing and formatting? AFAICS `c/storage/drivers/vfs/Init` ignores `options` entirely, so adding say `{"unused1", "unused2"}` would work for now—OTOH it might break anytime if `vfs` starts to parse the options.